src/channel.cr

```diff
@@ -14,10 +14,34 @@ abstract class Channel(T)
     end
   end
 
+  # A Fiber wrapper which executes a callback
+  # before being resumed or enqueued
+  # :nodoc:
```

Comment:
This should be moved into the first line to make it work (with blank line in between plz).

Reply:
resolved